### ISSUE TITLE
Stupid Idea I have

### ISSUE THREAD

**Issue by navanchauhan** [NullArray/AutoSploit#38]
What about giving a web interface for this project, so it could be launched via the internet? I have some free servers if they would help?

**Comment by NullArray** [comment]
It's not a stupid idea. But it's pretty easy to use the tool from the command line as is. Adding a web interface would mean a lot of work for minimum gains in usability, in my opinion.

**Comment by Souhardya** [comment]
It's even as of now is a nuclear bomb for script kiddies and adding a web interface would mean a whole lot more bunch of nut jobs causing wreck

**Comment by Ekultek** [comment]
@Souhardya we could create a web interface using Node.js and have an API calling the functions. Whatcha think?

**Comment by Souhardya** [comment]
Yeah we could in reference to the betterment of project it'll be cool

**Comment by NullArray** [comment]
Calling all Node.js devs? I don't really have any experience, coding in Node, but if anyone would be willing to help out with a web interface that would be great. Pull Requests are welcomed and very much appreciated in this regard.

**Comment by Ekultek** [comment]
I’ll do it in Node!

**Comment by NullArray** [comment]
Reliable as always @Ekultek. Appreciate it, dude.

**Comment by thehappydinoa** [comment]
@NullArray and @Ekultek I have a pretty solid background with design and api models lmk if I can do anything to help this idea.

**Comment by Ekultek** [comment]
@thehappydinoa if you can make an API, go for it. But make sure it has a web interface that it talks to as well

**Comment by thehappydinoa** [comment]
@Ekultek @NullArray ok cool, I would probably make it in Flask if I had the option, any opinions?

**Comment by Ekultek** [comment]
@thehappydinoa nah flask should be fine. Idk why you’d use it over django for a web app, but I can see it for the API I guess. Do what you do. The API needs to have credentials as well

**Comment by thehappydinoa** [comment]
@Ekultek tbh I'm just more familiar with, but if you like Django more I'll learn.

**Comment by an4cr0n** [comment]
I think Django is overkill for a simple web interface, IMO Node.js seems
more appropriate for something like Autosploit

Il 8 mar 2018 2:56 PM, "Aidan Holland" <notifications@github.com> ha
scritto:

**Comment by Ekultek** [comment]
@an4cr0n yes it does but I don’t have the time to write t right now

**Comment by thehappydinoa** [comment]
Ok,just lmk

**Comment by Ekultek** [comment]
Just do what you do

**Comment by thehappydinoa** [comment]
Cool

**Comment by NullArray** [comment]
Hey @thehappydinoa, if you want to go ahead and make a web front end in Flask, please feel free to do so. Didn't i read on twitter you were looking for a project? As always contributions to the AutoSploit project are very much appreciated and adding this type of functionality adds a whole new dimension in terms of user friendliness.

Also, like you've read on Twitter, i am doing some preliminary work with regards to the interactive RTFM project that you mentioned you'd like to be a part of as well. While i lay out the foundations for that project i think developing a web interface for AutoSploit in the mean time might be right up your alley. 

No obligations of course, just felt like mentioning it after the short conversation we had on Twitter about OSS projects.

**Comment by nsuchy** [comment]
@NullArray I have plenty of experience in Node and Express.js are you still interested in a Node.js WebUI?

**Comment by Ekultek** [comment]
@nsuchy yes we are. If you want to feel free.
